lib/shared/bucketing/src/bucketing.ts

```diff
@@ -105,7 +105,7 @@ export const getCurrentRolloutPercentage = (
             : null)
 
     if (!currentStage) {
-        return 0
+        return start
```

Comment:
pretty sure this just introduces a bug where it puts everyone in the start percentage of the rollout before the start date is actually triggered.